client/i18n/locales/english/translations.json

```
@@ -62,7 +62,8 @@
     "verify-email": "Verify Email",
     "submit-and-go": "Submit and go to next challenge",
     "go-to-next": "Go to next challenge",
-    "ask-later": "Ask me later"
+    "ask-later": "Ask me later",
+    "hide-project-preview": "Get started"
```

I am not a fan of the keys being so different to the values. I am not even really a fan of this suggestion. So, if anyone has better, please suggest away.

```suggestion
    "start-practice-project": "Get started"
```

---

Why not just `get-started`?

---

We already have a similar one.

---

We have `profile.get-started`, which is "Get started here."

This one would be `buttons.get-started`.

Alternatively, we could change the button text to something like `start-coding: Start Coding!`.

---

I like the `start-coding` one 👍